Makefile

```diff
@@ -513,7 +509,7 @@ VELERO_VERSION ?= $(shell go mod edit -json | jq -r '.Require[] | select(.Path =
 VELERO_BACKUP_NAME ?= velero.io_backups
 VELERO_BACKUP_CRD ?= $(EXTERNAL_CRD_DIR)/$(VELERO_BACKUP_NAME)-$(VELERO_VERSION).yaml
 
-SVELTOS_VERSION ?= v$(shell grep 'appVersion:' $(PROVIDER_TEMPLATES_DIR)/projectsveltos/Chart.yaml | cut -d '"' -f 2)
+SVELTOS_VERSION ?= v$(shell grep 'appVersion:' $(PROVIDER_TEMPLATES_DIR)/projectsveltos/Chart.yaml | cut -d ' ' -f 2)
```

Comment:
The `SVELTOS_VERSION` variable is now used solely to fetch the sveltos external CRDs to bin/crd. I believe it would be more appropriate to use the version of the `projectsveltos-crds` dependency rather than the `appVersion` of the `projectsveltos` chart. What do you think?